server/src/main/java/io/spine/server/BoundedContext.java

```diff
@@ -82,19 +81,36 @@
 @SuppressWarnings({"OverlyCoupledClass", "ClassWithTooManyMethods"})
 public abstract class BoundedContext implements Closeable, Logging {
 
+    /** Basic features of the context. */
     private final ContextSpec spec;
+
+    /** The Command Bus of this context. */
     private final CommandBus commandBus;
```

Comment:
Do we really need this kind of documentation? We give no new information to the reader.

Reply:
I added this as a sort of “warm-up” for the reader. The class is long. It simply prepares the mental context of a programmer who sees this code for the first time.